tests/Feature/MovieCreateTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieCreateTest extends TestCase
+{
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_movie_create_screen_requires_authorized_user()
+    {
+        $user = User::factory()->make();
+
+        $this->actingAs($user)
+        ->get('/movies')
+        ->assertStatus(200);
+    }
+
+    public function test_movie_create_new_movie_can_be_added() // No fails
+    {
+        $user = User::factory()->make();
+
+        $this->actingAs($user)
+        ->post('/movies', [
+            'title' => 'A Test movie',
+            'release_date' => '06/12/2031',
+            'poster' => 'https://m.media-amazon.com/images/M/MV5BMWZmYTI4MDctMzU4OC00ODJmLTkwMTgtYjRmMDRkMzc3NWZkXkEyXkFqcGdeQXVyNTAyODkwOQ@@._V1_.jpg',
+            'trailer' => 'https://www.youtube.com/watch?v=-1dSY6ZuXEY',
+            'blurb' => 'This is a sample blurb for the movie A Test movie'
+        ])->assertSessionHasNoErrors()
+        ->assertRedirect();
```

Comment:
When testing if the new movie is created, we need to also check if the database has that record :D 

You can use the $this->assertDatabaseHas()....

Reply:
a test has been added implementing assertDatabaseHas(). It only checks for one item (that being one of the possible trailers) that exists. Need to check if I can check for all columns

i.e

`$this->assertDatabaseHas('movies', [
            'title' => ....
            'release_date'=> ....
            'poster'=>.......
            'trailer' => 'https://www.youtube.com/embed/LIfcaZ4pC-4'
             etc
        ]);`